lyrics.py

```diff
@@ -11,6 +11,7 @@
 
 
 def get_lyrics(song_name):
+    song_name = re.sub(r'[\(\[].*?[\)\]]', '', song_name)
```

Comment:
This doesn't only remove the brackets, but everything written in the brackets as well. Did you intent this, and if you did, then what is the reasoning behind it?

Reply:
yes i did.

Say youre in spotify top 50 or something and youre listening to, for example, this song: Finesse (Remix) [feat. Cardi B]. (this is a good example because it contains both brackets and parentesis). 

If you search it on genius with the 'remix' text which isnt specific to this one song or the 'feat Cardi B' text which also isnt specific to this song, then the search becomes more likely to fail then if you just searched Finesse.

since the text in parentesis and brackets is usually only extra text with other musician names i just cut it from the search to improve the results. does that make sense?

Reply:
Yeah now I get what it's for! And it actually should improve my hacky genius link check, too. I'll test this a bit and if it works out, I'll merge it :)